packages/astro/src/core/config/vite-load.ts

```diff
@@ -6,6 +6,7 @@ import { debug } from '../logger/core.js';
 
 async function createViteServer(root: string, fs: typeof fsType): Promise<ViteDevServer> {
 	const viteServer = await createServer({
+		configFile: false /* prevent vite from trying to discover a vite conf file */,
```

Comment:
```suggestion
		configFile: false,
```
I don't think we need an explanation here, I think it's clear what the intent is.